metadata.json

```diff
@@ -87,6 +87,7 @@
     }
   ],
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">= 2.4.0 < 5.0.0"}
+    {"name":"puppetlabs/stdlib","version_requirement":">= 2.4.0 < 5.0.0"},
+    {"name":"puppet-community/puppet-archive","version_requirement":"0.3.0"}
```

Comment:
Adding this dependency would be backwards incompatible, and not needed if users are not using java::oracle, so perhaps it should just be documented as an optional dependency for when using java::oracle instead of a hard dependency.